### PR TITLE
Adding Labels to Contact Us Minicons

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Adding labels to the contact us minicons. 

## Additions

- Added labels to the contact us minicons

## Changes

- Moved the phone numbers, email address to their own paragraph instead of a span.
- Moved the address to the same unordered-icon-list format as the rest of the contact information.

## Testing

- /contact-us/

## Review

- @jimmynotjim 
- @anselmbradford 
- @sebworks 

## Preview

image

image

## Notes
- My first pull request for cfgov-refresh :facepunch:

### CODE REVIEW

**Comment by jimmynotjim** [comment]
- Use one label above multi-contact sources (ex: two phone numbers)
- Add label above phone numbers in "Submit a Complaint" section and update vanity phone numbers
- Add tel links to phone numbers

**Comment by KimberlyMunoz** [comment]
Fixed the labels for multiple contact sources and added support for tel: links based on what I saw in the footer. Basically the tel: links are there, but only visible on smaller screens.

For the vanity numbers in Submit a Complaint, there's only two fields for phone numbers. I'm going to do a simple replace to add the vanity numbers and so that the end user only needs to update the number and description.

**Comment by KimberlyMunoz** [comment]
Content is ramped up. Here's how it looks now and it's ready for a second round of reviews.

image

**Comment by jimmynotjim** [comment]
Looking good. One last thing is to add a "Phone:" label with an icon above the "Submit a complaint" numbers to match the "Inquiries" one.

**Comment by KimberlyMunoz** [comment]
Thanks for feedback! Added.

**Comment by KimberlyMunoz** [comment]
There was an outstanding issue where we discovered the field names for phone numbers and email were wrong. I went ahead and updated them since it makes this PR easier to review.

File: contact-us/submit-a-complaint.html [hunk 1]
```diff
@@ -7,25 +7,34 @@ <h1 class="h4">
...
+                    <span class="micro-copy micro-copy__large">
+                        {{ submit_a_complaint.phone_0_desc }}
+                    </span>
+                {%- endif %}
+            </a>
+        </p>
+        {%- if submit_a_complaint.phone_1_num %}
+        <p>
+            <a class="list_link list_link__phone" href="tel:{{ submit_a_complaint.phone_1_num }}">
+                    {{ format_phone(submit_a_complaint.phone_1_num)| replace("2372", "CFPB (2372)") }}
```
Comment (anselmbradford) [inline contact-us/submit-a-complaint.html]: Very minor:
This line has an extra indent.

**Comment by anselmbradford** [comment]
This looks good to me :+1: Could you add the changes to the `Unreleased` section in the changelog please? Otherwise, GTG from my end. Might want another check off from @jimmynotjim before squash 'n' merge.

File: contact-us/submit-a-complaint.html (hunk 1 above)
Comment (KimberlyMunoz) [inline contact-us/submit-a-complaint.html]: L25? Fixing!

Comment (anselmbradford) [inline contact-us/submit-a-complaint.html]: L28 https://github.com/cfpb/cfgov-refresh/blob/DF-1803-contact-us-minicons/contact-us/submit-a-complaint.html#L28

Comment (KimberlyMunoz) [inline contact-us/submit-a-complaint.html]: I didn't see that one at all! Thanks for the catch. Fixed!

**Comment by KimberlyMunoz** [comment]
What do you think, @jimmynotjim?

**Comment by jimmynotjim** [comment]
Tested locally and running great. Thanks for including the indexing fix in the contact list as well. :+1:

**Pull request merged**